### ISSUE TITLE
auth().sendSignInLinkToEmail crashes iOS app

### ISSUE THREAD

**Issue by IljaDaderko** [invertase/react-native-firebase#1105]
### Issue

I have implemented passwordless sign in (first step) via function below

  ```
emailSignIn = async (email: string) => {
    try {
      await auth().sendSignInLinkToEmail(email, actionCodeSettings)
      Alert.alert('Success', `Email sent to ${email}`)
    } catch (e) {
      Alert.alert('Error', e.message)
    }
  }
```

When I call this function (checked that it is not alerts causing the issue and that correct email is passed) my ios app crashes, with following screen displayed in xcode

<img width="1440" alt="screen shot 2018-05-19 at 14 08 08" src="https://user-images.githubusercontent.com/3154053/40268014-98f6eff4-5b6e-11e8-9182-17c053d3745f.png">

I decided to further test this and executed request with empty email, it went through fine and I got related firebase error, so this has to be something with success flow?

### Environment

1. Application Target Platform: iOS

2. Development Operating System: macOS High Sierra

3. Build Tools: Xcode 9.3, iOS SDK 5.0.1

4. React Native version: 0.55.4

5. RNFirebase Version: 4.2.0

6. Firebase Module: Auth

**Comment by rikur** [comment]
Not so sure if it was on your side.

**Comment by IljaDaderko** [comment]
@rikur yep it turned out that way, I have this implemented and working now, also make sure you use `auth().` and not `auth.`

**Comment by cltsang** [comment]
For future fellows, my app crashes when handleCodeInApp is not set to true in actionCodeSettings.
